src/test/java/org/jenkinsci/plugins/casc/core/AdminWhitelistRuleConfiguratorTest.java

```diff
@@ -42,7 +43,7 @@
     public void checkM2SSecurityKillSwitch_enabled() throws Exception {
         final Jenkins jenkins = Jenkins.getInstance();
         AdminWhitelistRule rule = jenkins.getInjector().getInstance(AdminWhitelistRule.class);
-        Assert.assertTrue("MasterToSlave Security should be enabled", rule.getMasterKillSwitch());
+        Assert.assertFalse("MasterToSlave Security should be enabled", rule.getMasterKillSwitch());
```

Comment:
Don't we need to switch context now?
Shouldn't it actually say should be disabled? and method should be called disabled? 😁
@oleg-nenashev

Reply:
@casz I'm now confused :P should I merge it or do you have doubts?

Reply:
Then you know how I felt trying to figure it out. 😀 But the test context is Master To slave I believe we have to switch and that would also make sense since we are inverting :)